### DISCUSSION TITLE
Enable Edit Quote after Status Accepted

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#15503] [How to] [answered]
Hi,

How do I enable user to change status of Quote after accepted.

Currently record is locked for edit after it is accepted.

![image](https://user-images.githubusercontent.com/12723595/144229957-9360619d-4079-4ba8-9df2-485a7d7cc592.png)

Regards
Jeevan Pendhari

**Comment by mariuszkrzaczkowski** [comment]
There is no lock by default, there are several ways to do it, check it
![image](https://user-images.githubusercontent.com/10198654/144231133-75576de3-8be7-4aa1-aff8-5fb4f847e858.png)

**Comment by rskrzypczak** [comment, marked as answer]
check your workflow
![image](https://user-images.githubusercontent.com/10221999/144232539-a312620d-e0c1-4d59-9f50-263773065f14.png)

**Reply by JEEVAN28** [reply]
This worked, thanks for reply
